languagetool-gui-commons/src/main/java/org/languagetool/gui/Tools.java

```diff
@@ -205,6 +202,11 @@ public static void configureFromRules(JLanguageTool langTool, Configuration conf
         langTool.disableCategory(new CategoryId(categoryName));
       }
     }
+    if(config.getEnabledRulesOnly()) {
```

Comment:
Is this an unrelated bugfix?

Reply:
I forget to mention that in the summary: I have added the possibility to only enable the rules which are explicitly mentioned in the configuration file. Afaik that was not possible before. I'm using that for the demo page where only neural network rules are enabled.